### ISSUE TITLE
gnuplot: Library not loaded: /usr/local/opt/qt/lib/QtNetwork.framework/Versions/5/QtNetwork

### ISSUE THREAD

**Issue by ryofurue** [Homebrew/homebrew-core#54327]
# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.

- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

## What you were trying to do (and why)
I was launching `gnuplot`.

## What happened (include command output)
gnuplot doesn't run because dynamic lib `QtNetwork` cannot be loaded. See below for details.
<details>
  <summary>Command output</summary>
  <pre>
$ type gnuplot
gnuplot is /usr/local/bin/gnuplot
$ gnuplot
dyld: Library not loaded: /usr/local/opt/qt/lib/QtNetwork.framework/Versions/5/QtNetwork
  Referenced from: /usr/local/bin/gnuplot
  Reason: image not found
fish: 'gnuplot' terminated by signal SIGABRT (Abort)
$
 </pre>
</details>

## What you expected to happen

N/A

## Step-by-step reproduction instructions (by running `brew install` commands)
````
$ brew install gnuplot
$ gnuplot
````
## `brew gist-logs`
results in error
````
Error: No logs.
````
## Output from `brew config`
````
HOMEBREW_VERSION: 2.2.15-18-g61b98a5
ORIGIN: https://github.com/Homebrew/brew
HEAD: 61b98a5e2ed3d8d00ae93ae325a8fe5363ae2ac2
Last commit: 2 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d234fde7164f11fce41fcfedd2efe5a341fd553a
Core tap last commit: 5 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.PsOz2VojhG/org.macosforge.xquartz:0
HOMEBREW_MAKE_JOBS: 8
CPU: octa-core 64-bit kabylake
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
Clang: 11.0 build 1103
Git: 2.26.2 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.4-x86_64
CLT: 11.4.1.0.1.1586360307
Xcode: N/A
XQuartz: 2.7.11 => /opt/X11
````

## Output from `brew doctor`
````
Your system is ready to brew.
````

**Comment by Bo98** [comment]
Sound like something is broken with ypur `qt` install. `gnuplot` depends on `qt`.

**Comment by ryofurue** [comment]
> Sound like something is broken with ypur qt install.

Bingo!  Thanks!

After reading your comment, I first did
````
brew reinstall qt
````
This issued a lot of error messages like `cp: /usr/local/Cellar/qt/. . . InputMethodEvent: Permission denied`.  So, I then went to `/usr/local/Cellar` and did `sudo chmod -R . . . `.

Then `gnuplot` started to work!  I've never changed the permissions under `/usr/local/Cellar` before this.  That means, some of the homebrew packages I directly or indirectly installed did it?
